### PR TITLE
First stab at renewed charter

### PR DESCRIPTION
**Pull request by tidoust**
This creates a first draft of a renewed charter for the Media & Entertainment IG.

The pull request is divided into 3 commits to ease review:

- First commit adds the current IG charter as basis for the work
- Second commit integrates wording changes introduced in the charter template in the last couple of years
- Third commit refreshes slightly the list of groups in the coordination list

No further changes at this stage.

Linked to #43.

### CODE REVIEW

File: charters/charter-2021.html [hunk 1]
```diff
@@ -0,0 +1,841 @@
...
+            <dd>The Second Screen Community Group, companion group of the Second Screen Working Group, discusses the development of an Open Screen Protocol to improve the interoperability between first and second screens.</dd>
+
+            <dt><a href="https://www.w3.org/community/wicg/">Web Platform Incubator CG</a></dt>
+            <dd>The Web Platform Incubator Community Group incubates a number of proposals from the community, including media-relevant ideas such as <a href="https://github.com/WICG/canvas-color-space">Canvas Color Spaces</a>, <a href="https://wicg.github.io/video-rvfc/">HTMLMediaElement.requestVideoFrameCallback()</a>, and <a href="https://wicg.github.io/media-feeds/">Media Feeds</a>.</dd>
+
+            <dt><a href="https://www.w3.org/community/webmediaapi/">Web Media API CG</a></dt>
+            <dd>The Web Media API Community Group incubates a minimum set of web technologies that developers of media web applications can rely on being supported across devices. The group was proposed by and continues to cooperate with the <a href="https://standards.cta.tech/kwspub/wave/">CTA WAVE Project</a>.</dd>
```
Comment (JohnRiv) [inline charters/charter-2021.html:394]: Cool URLs don't change, but unfortunately's CTA WAVE's URL has:
```suggestion
            <dd>The Web Media API Community Group incubates a minimum set of web technologies that developers of media web applications can rely on being supported across devices. The group was proposed by and continues to cooperate with the <a href="https://cta.tech/WAVE">CTA WAVE Project</a>.</dd>
```

File: charters/charter-2021.html [hunk 2]
```diff
@@ -0,0 +1,841 @@
...
+
+            <dt><a href="https://www.cta.tech/">CTA</a></dt>
+            <dd>The Consumer Technology Association is the trade
+                organization for the consumer electronics industry,
+                which promotes the consumer technology industry
+                through technology policy, events, research, promotion
+                and the fostering of business and strategic
+                relationships. CTA organizes the annual Consumer
+                Electronics Show (CES), and hosts the 
+                <a href="https://standards.cta.tech/kwspub/wave/">CTA
```
Comment (JohnRiv) [inline charters/charter-2021.html:442]: One more WAVE URL update
```suggestion
                <a href="https://cta.tech/WAVE">CTA
```

**Review by JohnRiv — COMMENTED** [review]
I noticed the URLs for CTA WAVE should be updated

**Review by chrisn — APPROVED** [review]
Looks good, thank you!

**Pull request merged**